ext/session/session.c

```diff
@@ -1409,36 +1409,36 @@ static zend_result php_session_send_cookie(void)
 		t = tv.tv_sec + PS(cookie_lifetime);
 
 		if (t > 0) {
-			date_fmt = php_format_date("D, d M Y H:i:s \\G\\M\\T", sizeof("D, d M Y H:i:s \\G\\M\\T")-1, t, 0);
-			smart_str_appends(&ncookie, COOKIE_EXPIRES);
+			date_fmt = php_format_date(ZEND_STRL("D, d M Y H:i:s \\G\\M\\T"), t, false);
+			smart_str_appendl(&ncookie, ZEND_STRL(COOKIE_EXPIRES));
```

Comment:
For cases like these, I prefer the original code. The strlen() call will almost certainly be evaluated at compile time.

Reply:
Right, I forgot that this is inlined. I was expecting the need for a `_literal` suffix API.